client/blocks/reader-post-options-menu/blog-sticker-menu-item.jsx

```diff
@@ -18,12 +18,9 @@ class ReaderPostOptionsMenuBlogStickerMenuItem extends React.Component {
 		hasSticker: React.PropTypes.bool,
 	};
 
-	addSticker = () => {
-		if ( this.props.hasSticker ) {
-			return null;
-		}
-
-		this.props.addBlogSticker( this.props.blogId, this.props.blogStickerName );
+	toggleSticker = () => {
+		const action = this.props.hasSticker ? 'removeBlogSticker' : 'addBlogSticker';
```

Comment:
It's definitely neater :) There's one problem though - the linter complains about:

`'addBlogSticker' is already declared in the upper scope`

If we end up aliasing the action name to get around that, it suddenly doesn't feel as neat. What do you think?